data/json/items/comestibles/med.json

```diff
@@ -1186,16 +1169,69 @@
         "id" : "vitamins",
         "type" : "COMESTIBLE",
         "comestible_type" : "MED",
-        "name" : "vitamin",
-        "description" : "Essential dietary nutrients conveniently packaged in pill form.  Take frequently to help regulate your body's health and immune functions.",
+        "name" : "multivitamin",
+        "description" : "Essential dietary nutrients conveniently packaged in pill form.  An option of last resort when a balanced diet is not possible.  Excess use can cause hypervitaminosis.",
         "weight" : 1,
         "volume" : 1,
         "price" : 500,
+        "price_postapoc": 5000,
         "charges" : 20,
-        "stack_size" : 100,
+        "stack_size" : 20,
         "symbol" : "!",
         "color" : "magenta",
-        "use_action" : "VITAMINS"
+        "use_action" : {
+            "type" : "consume_drug",
+            "activation_message" : "You take some multivitamins.",
+            "vitamins": [
+                [ "calcium", 25, 50 ],
+                [ "iron", 25, 50 ],
+                [ "vitA", 25, 50 ],
+                [ "vitB", 25, 50 ],
+                [ "vitC", 25, 50 ]
+            ]
+        }
+    },
+    {
+        "id" : "inj_vitb",
+        "type" : "COMESTIBLE",
+        "comestible_type" : "MED",
+        "name" : "injectable vitamin B",
+        "description" : "Small vials of pale yellow liquid containing soluble vitamin B for injection.",
+        "weight" : 40,
+        "volume" : 1,
+        "price" : 150,
+        "price_postapoc": 8000,
```

Comment:
It certainly isn't worth more than a whole stack of multivitamins. Especially not after apocalypse.